src/i_system.c

```diff
@@ -150,7 +155,11 @@ void I_SafeExit(int rc)
     }
   }
 
-  exit(rc);
+  #ifdef WIN_LAUNCHER
```

Comment:
What is this for?

Reply:
`exit()` does not work on Windows when called from a DLL. On Windows have the woof.exe and woof.com launcher programs, and all the code is in woof.dll.

Reply:
I see, thanks!